scripts/build-data.mjs

```diff
@@ -40,12 +40,12 @@ Missing locales are ignored, react-intl will use the default messages for them.
  */
 import * as fs from 'fs';
 import * as path from 'path';
-import {sync as mkdirpSync} from 'mkdirp';
+import mkdirp from 'mkdirp';
```

Comment:
Since `mkdirp` is in CommonJS format, Node.js didn't support this import as it was previously phrased.